code/game/gamemodes/cult/runes.dm

```diff
@@ -399,14 +414,21 @@ var/list/teleport_runes = list()
 	if(!offering)
 		rune_in_use = 0
 		return
-	if(((ishuman(offering) || isrobot(offering)) && offering.stat != DEAD) || is_sacrifice_target(offering.mind)) //Requires three people to sacrifice living targets
+	if(((ishuman(offering) || isrobot(offering)) && offering.stat != DEAD) || is_sacrifice_target(offering.mind))
 		if(invokers.len < 3)
 			for(var/M in invokers)
 				to_chat(M, "<span class='cultitalic'>[offering] is too greatly linked to the world! You need three acolytes!</span>")
 			fail_invoke()
-			log_game("Sacrifice rune failed - not enough acolytes and target is living")
+			log_game("Sacrifice rune failed - not enough acolytes.")
 			rune_in_use = 0
 			return
+	if(offering.stat == DEAD)
```

Comment:
Ah, this raises 2 issues:
First, what if a sacrifice target dies? Does this leave the objective uncompleteable?
Second, isn't sacrificing the dead a means for a cult to bring itself back from the brink? Using this, a single cultist can go from just themself, to 2 shades and themself, sufficient to do full conversions.